File: leadera/src/main/java/com/leadera/leadera/dto/LeadsHoyResponse.java
package com.leadera.leadera.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LeadsHoyResponse {

    private List<LeadHoyDTO> nuevosSinContacto;
    private List<LeadHoyDTO> prioritarios;
    private List<LeadHoyDTO> seguimientosDeHoy;

    private List<LeadHoyDTO> contactadosHoy;

    private int totalTareasDelDia;
    private int tareasCompletadasDelDia;

    private int totalPrioritarios;
    private int totalNuevos;
    private int totalSeguimientos;

}

File: leadera-front/src/app/core/models/leads-hoy-response.ts
import { LeadHoy } from "./lead-hoy";

export interface LeadsHoyResponse {
    nuevosSinContacto: LeadHoy[];
    prioritarios: LeadHoy[];
    seguimientosDeHoy: LeadHoy[];
    contactadosHoy: LeadHoy[];
    totalTareasDelDia: number;
    tareasCompletadasDelDia: number;
    totalPrioritarios: number;
    totalNuevos: number;
    totalSeguimientos: number;
}
